test/cases/oscap.sh

```diff
@@ -362,6 +362,7 @@ clean_up
 ###############################
 
 # Write a blueprint for hardened image.
+# TODO: Remove firewalld rules from tailoring once https://github.com/ComplianceAsCode/content/issues/11275 is fixed
```

Comment:
Do you have a COMPOSER ticket for this? I think this TODO will get forgotten very easily.

Reply:
COMPOSER-2076